portlayer/storage/image.go

```diff
@@ -34,6 +34,9 @@ type Image struct {
 	Parent *url.URL
 
 	Store *url.URL
+
+	// Metadata associated with the image.
+	Metadata map[string][]byte
```

Comment:
Is there a expectation of using different keys for sending the metadata or should imagec create a key (let's say docker) and dump the json as a []byte there?